src/app.rs

```diff
@@ -3763,13 +3767,22 @@ pub fn remove_tag(task: &mut Task, tag: &str) {
 }
 
 #[cfg(test)]
```

Comment:
I took the liberty of applying some outstanding clippy suggestions.

Most of these should be negligible, correct me if I'm wrong.

Two are imo controversial but justifiable, number one is this lint.
Context:
let shell = message.as_str().replace("'", "\'");
triggers the clippy warning for a single char pattern, it's suggestion ''' is imo more confusing than helpful.